docker-hub/builds/advanced.md

```diff
@@ -50,9 +50,8 @@ basic `docker` commands, so you must include a similar build, test or push
 command in the hook or your automated process does not complete.
 
 To override these phases, create a folder called `hooks` in your source code
-repository at the same directory level as your Dockerfile. Create a file called
-`hooks/build`, `hooks/test`, or `hooks/push` and include commands that the
-builder process can execute, such as `docker` and `bash` commands (prefixed appropriately with `#!/bin/bash`).
+repository at the same directory level as your Dockerfile. Create a script called
+`hooks/build`, `hooks/test`, or `hooks/push` and give it execution privileges. This can be a shell, Perl or Python script, or anything that can be executed by the Linux based runner.
```

Comment:
```suggestion
`hooks/build`, `hooks/test`, or `hooks/push` and give it execution privileges. This can be a shell, Perl or Python script, or anything that can be executed by the Linux-based runner.
```